src/main/java/seedu/linkedout/model/applicant/Applicant.java

```diff
@@ -164,4 +164,24 @@ public String toString() {
         }
         return builder.toString();
     }
+
+    /**
+     * Returns integer value representing if String value of name of current applicant
+     * is more or less, or equal String value of name of current applicant to {@code applicant2}.
+     */
+    public int compareNames(Applicant applicant2) {
+        String nameOfFirstApplicant = this.getName().toString().toLowerCase();
```

Comment:
I do not believe this belongs here, because Applicant is a data class. 
Perhaps, instead comparison should be made in their own classes.
E.g get Name to implement compareTo, and put this method there.

The integrated logic of this method such as getting the names of the applicant should be done in the comparator. Comparison of names should be written in a method in the Name class.

Apply above logic to the compareJobs method as well.

Reply:
Thanks, this was helpful!